src/Tribe/Assets.php

```diff
@@ -83,17 +83,17 @@ public function register() {
 		);
 
 		// All tribe events pages
-		tribe_asset(
-			$plugin,
-			'tribe_events-admin',
-			'events-admin.css',
-			array(),
-			'admin_enqueue_scripts',
-			array(
-				'groups'       => array( 'events-admin' ),
-				'conditionals' => array( $this, 'should_enqueue_admin' ),
-			)
-		);
+//		tribe_asset(
+//			$plugin,
+//			'tribe_events-admin',
+//			'events-admin.css',
+//			array(),
+//			'admin_enqueue_scripts',
+//			array(
+//				'groups'       => array( 'events-admin' ),
+//				'conditionals' => array( $this, 'should_enqueue_admin' ),
+//			)
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected at least 2 tabs, found 0
* :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: 03d40dc303af6bbc27b2f4ead4563f48</sub>